doc/nrf/releases_and_maturity/releases/release-notes-2.8.0.rst

```diff
@@ -0,0 +1,1550 @@
+
+nRF RPC libraries
+-----------------
+
+* Added:
+
+  * An experimental serialization of Openthread APIs.
+  * The logging backend that sends logs through nRF RPC events.
+
+* Updated the internal Bluetooth serialization API and Bluetooth callback proxy API to become part of the public NRF RPC API.
+
+Other libraries
+---------------
+
+* Added the :ref:`nrf_compression` library with support for the LZMA decompression.
+
+* :ref:`lib_date_time` library:
+
+  * Added:
+
+    * A retry feature that reattempts failed date-time updates up to a certain number of consecutive times.
+    * The Kconfig options :kconfig:option:`CONFIG_DATE_TIME_RETRY_COUNT` to control whether and how many consecutive date-time update retries may be performed, and :kconfig:option:`CONFIG_DATE_TIME_RETRY_INTERVAL_SECONDS` to control how quickly date-time update retries occur.
+
+  * Fixed a bug that caused date-time updates to not be rescheduled under certain circumstances.
+
+* :ref:`lib_ram_pwrdn` library:
+
+  * Added support for the nRF54L15 SoC.
+
+* :ref:`dult_readme` library:
+
+  * Added DULT motion detector feature.
+    You can enable this feature using the :kconfig:option:`CONFIG_DULT_MOTION_DETECTOR` Kconfig option.
+
+Security libraries
+------------------
+
+* :ref:`nrf_security_readme` library:
+
+  * Added the :kconfig:option:`CONFIG_PSA_WANT_ALG_SP800_108_COUNTER_CMAC` Kconfig option to key derivation function configurations in :ref:`nrf_security_driver_config`.
+    The Kconfig option enables support for the derivation function SP 800-108r1 CMAC in counter mode, which is supported by the nrf_cracen driver.
+
+* :ref:`trusted_storage_readme` library:
+
+  * Added support for Zephyr Memory Storage (ZMS), as an alternative to the NVS file system.
+
+sdk-nrfxlib
+-----------
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Scripts
+=======
+
+This section provides detailed lists of changes by :ref:`script <scripts>`.
+
+* :ref:`nrf_desktop_config_channel_script` script:
+
+  * Added semantic version support for devices that use the :ref:`SUIT DFU <ug_nrf54h20_suit_dfu>`.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``a4eda30f5b0cfd0cf15512be9dcd559239dbfc91``, with some |NCS| specific additions.
+
+The code for integrating MCUboot into |NCS| is located in the :file:`ncs/nrf/modules/mcuboot` folder.
+
+The following list summarizes both the main changes inherited from upstream MCUboot and the main changes applied to the |NCS| specific additions:
+
+* Added:
+
+  * :ref:`Compressed firmware update support <mcuboot_image_compression>`.
+  * Support for SHA-512.
+  * Support for ED25519 'PURE' signature.
+    See the :kconfig:option:CONFIG_BOOT_SIGNATURE_TYPE_PURE Kconfig option in MCUboot for details.
```

Comment:
Format for the Kconfig is wrong. But I don't  think we can link anyway. Since it cannot be linked, may be we should rephrase as  (Related Kconfig - ``CONFIG_BOOT_SIGNATURE_TYPE_PURE``)